vita3k/gui/src/settings_dialog.cpp

```diff
@@ -251,6 +259,12 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Select your preferred cpu backend.");
         ImGui::Spacing();
+        if (config.cpu_backend == "Dynarmic") {
```

Comment:
Is good but maybe hetter with using enum here
```suggestion
        if (config_cpu_backend == CpuBackend:: Dynarmic) {
```